pylint_odoo/checkers/modules_odoo.py

```diff
@@ -822,7 +822,8 @@ def _is_replaced_field(view):
             return None
         replaces = \
             arch.xpath(".//field[@name='name' and @position='replace'][1]") + \
-            arch.xpath(".//xpath[@position='replace'][1]")
+            arch.xpath(".//xpath[@position='replace'][1]") + \
```

Comment:
Is this required?

AFAIK thanks to the below line, this one is already considered

Reply:
Done